test.js

```diff
@@ -215,5 +215,5 @@ test('prevent setting/getting `__proto__`', t => {
 });
 
 test('return default value if path is invalid', t => {
-	t.is(dotProp.get({}, '__proto__', '🦄'), '🦄');
+	t.is(dotProp.get({}, 'nonexistent', '🦄'), '🦄');
```

Comment:
The path is only "invalid" if a disallowed property is passed. Testing for a legal property is already tested for.

Reply:
Is there any other non-legal value we could use that is not `__proto__`?

Reply:
https://github.com/sindresorhus/dot-prop/blob/09adad91e5a5072014b3380e769be8aae89738f2/index.js#L5-L7